### DISCUSSION TITLE
Scrollbar is not show, how to fix this?

### DISCUSSION THREAD

**Discussion by nickp-real** [hrsh7th/nvim-cmp#1628] [Q&A]
![image](https://github.com/hrsh7th/nvim-cmp/assets/69962410/dcbb8b07-fc1a-4594-a03a-513a2e29c770)
pmenusbar highlight
![image](https://github.com/hrsh7th/nvim-cmp/assets/69962410/e183b4b7-0469-4098-8da7-82e54b260db0)
[cmp config](https://github.com/NickP-real/.dotfile/blob/4b177afd20cb9f5e71e5adff246fda180fd8c147/.config/nvim/lua/plugins/coding.lua#L51-L285)
[onedarkpro config](https://github.com/NickP-real/.dotfile/blob/main/.config/nvim/lua/plugins/colorscheme.lua)

**Comment by fitrh** [comment]
The scrollbar only shows when the completion items are bigger than the height of the completion window, try setting your `'pumheight'` to a small enough value like 3

For me, with lua-language-server and `:set pumheight=5`, typing `s` gave me enough completion items to show the scrollbar

**Reply by nickp-real** [reply]
still doesn't work

**Reply by fitrh** [reply]
Make sure you have this

```lua
window = {
  completion = {
    scrollbar = true,
  },
}
```

Although I think it is the default

**Reply by fitrh** [reply]
And make sure you have a sufficient number of completion items to get it scrolling
